artist_query/lib/artist_query.ex

```diff
@@ -0,0 +1,41 @@
+defmodule ArtistQuery do
+  def main(_args) do
+    artist = prompt
+    load_songs("music.csv")
+    |> Enum.filter_map(&(String.upcase(&1["Artist"])) == artist, &(&1["Name"]))
+    |> put_songs(titlecase(artist))
+
+    main([]) # Repeat forever
+  end
+
+  defp put_songs(songs, artist) do
+    message = "* Songs by #{artist} *"
```

Comment:
I think displaying the banner could probably be pulled out into a shorter, well named method